app/models/concerns/digital_object/datacite.rb

```diff
@@ -71,7 +57,7 @@ def change_doi_status_to_unavailable
   # Uses the modify identifier API call, returns true if metadata update was successful
   # if not, returns false
```

Comment:
Does this method ever return false anymore?  Or does the comment need to be updated?

Reply:
Comment updated as suggested.